### PR TITLE
Update Multiclass tutorial for ML.NET V .10

### LINKED ISSUE #10390: Update for v0.10
This line is broken in v0.10 as the constructors have been made internal:
var trainer = new SdcaMultiClassTrainer(_mlContext,DefaultColumnNames.Label, DefaultColumnNames.Features);

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 77fc5686-2288-d607-8136-3146ed02dbf6
* Version Independent ID: 8a16b9e9-b5cb-a55e-2135-191bc62c1656
* Content: [Use ML.NET in a GitHub issue multiclass classification scenario](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification)
* Content Source: [docs/machine-learning/tutorials/github-issue-classification.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/github-issue-classification.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

### PR DESCRIPTION
**Pull request by JRAlexander**
Update multiclass tutorial topic for .10 release.
Fixes #10390

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification?branch=pr-en-us-10490)

### CODE REVIEW

**Review by luisquintanilla — CHANGES_REQUESTED** [review]
Looks good. Minor update - ChangeIDataView xref to Microsoft.Data.DataView.IDataView

**Review by luisquintanilla — APPROVED** [review]
